tests/helpers.py

```diff
@@ -90,3 +90,11 @@ def send(self, *args):
 
 
 _MockResponse = namedtuple('MockResponse', ['status_code', 'text', 'headers'])
+
+
+def assert_raises(exception_class, action):
+  try:
+    action()
+    raise AssertionError, "Should have raised an %s" % exception_class
```

Comment:
That should already happen because we only `except exception_class`. I tested it like so:

    def foo():
      def bar():
        raise TypeError, "baz"
      assert_raises(ArithmeticError, bar)
    assert_raises(TypeError, foo)